controllers/external-accounts.js

```diff
@@ -40,6 +43,38 @@ const getExternalAccountData = async (req, res) => {
       return res.boom.unauthorized("Token Expired. Please generate it again");
     }
 
+    const { id: userId, roles } = req.userData;
+
+    if (!roles.in_discord) {
```

Comment:
- can we move this inside a function and return the data required here? it will make it easy to test the logic that's written here.

Reply:
Okay I'll do that